packages/rancher-sriov/templates/crd-template/Chart.yaml

```diff
@@ -9,4 +9,4 @@ annotations:
   catalog.cattle.io/namespace: cattle-sriov-system
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/hidden: "true"
-  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/permits-os: linux
```

Comment:
did you remove support from tolerating a single Windows node in a cluster? So this can only be installed in a cluster with only Linux nodes?

`permits-os` == you will not fail with these OSs existing. i.e. you have nodeSelectors and tolerations

`os` == you deploy os specific components. Like rancher-monitoring deploying rancher-windows-exporters to specifically target Windows nodes.

Reply:
sriov in Kubernetes is a linux-only technology. So this is more fixing a bug